cmake/SSGCommon.cmake

```diff
@@ -422,8 +379,8 @@ macro(ssg_build_link_xccdf_oval_ocil PRODUCT)
         OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/xccdf-linked.xml"
         OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/oval-linked.xml"
         OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/ocil-linked.xml"
-        COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${SSG_BUILD_SCRIPTS}/relabel_ids.py" "${CMAKE_CURRENT_BINARY_DIR}/xccdf-unlinked.xml" ssg
-        DEPENDS generate-internal-${PRODUCT}-xccdf-unlinked.xml
+        COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${SSG_BUILD_SCRIPTS}/relabel_ids.py" "${CMAKE_CURRENT_BINARY_DIR}/xccdf-with-stig-references.xml" "${CMAKE_CURRENT_BINARY_DIR}/xccdf-linked.xml" ssg
+        DEPENDS generate-internal-${PRODUCT}-xccdf-with-stig-references.xml
```

Comment:
@matejak I expect a problem here because "xccdf-with-stig-references" has been removed by https://github.com/ComplianceAsCode/content/pull/8474/. So it probably needs to be replaced here by "shorthand.xml" Don't merge at this moment!